Apps/WebClient/src/Server/Controllers/BetaRequestController.cs

```diff
@@ -57,15 +53,17 @@ public class BetaRequestController
         /// Posts a beta request json to be inserted into the database.
         /// </summary>
         /// <returns>The http status.</returns>
+        /// <param name="authorization">The users access token.</param>
         /// <param name="betaRequest">The beta request model.</param>
         /// <response code="200">The beta request record was saved.</response>
         /// <response code="400">The beta request was already inserted.</response>
         /// <response code="401">The client must authenticate itself to get the requested response.</response>
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
         [HttpPut]
         [Authorize(Policy = UserPolicy.UserOnly)]
-        public IActionResult CreateBetaRequest([FromBody] BetaRequest betaRequest)
+        public IActionResult CreateBetaRequest([FromHeader] string authorization, [FromBody] BetaRequest betaRequest)
         {
+            // this.logger.logTrace(authorization);
```

Comment:
what is our best practice re. logging inputs in overall of this project? Should we log all the inputs at the beginning of each method?

Reply:
We don’t have logging in many of our controllers and we haven’t DJs used a standard.